src/main/webapp/app/shared/pages/imprint-page/imprint-page.component.scss

```
@@ -0,0 +1,35 @@
+.imprint-page {
+  color: var(--p-text-primary);
+  padding: 3rem 5rem 1.5rem 1.5rem;
+  max-width: 1200px;
+  margin-right: auto;
+  .headline {
+    font-size: 2.3rem;
+    font-weight: 700;
+  }
+
+  .subheadline {
+    font-size: 1.8rem;
+    font-weight: 700;
+    margin-bottom: 1rem;
+    margin-top: 2rem;
+  }
+
+  .text {
+    display: block;
```

Could you add this as well so the text looks more organized
```suggestion
    text-align: justify;
    display: block;
```